src/components/Progress.js

```diff
@@ -29,18 +36,18 @@ const Progress = observer(({store}) => (
           <CustomContentProgressBar>
             {/* TODO: Link the data to this number */}
             <Percentage>{total_treatment_days}</Percentage>
-
             {/* TODO style component */}
-            <div>
+            <TreatmentDays>
               <strong>Days of Treatment</strong>
-            </div>
+            </TreatmentDays>
           </CustomContentProgressBar>
         </ProgressBarContainer>
 
       <Divider />
         <Fold>
           <Question>
-            Medication History
+            {/* TODO: Figure out how to add vertical alignment for these icons */}
+          <Icon size="1.5rem" vertical-align="bottom" color={darkgrey} path={mdiPill} /> Medication History
           </Question>
```

Comment:
For vertical alignment, I would try to change these `Question` elements into `flexbox` layouts.
You might need to wrap the text in `<span>`s.
I think flexbox's `align-items` might work?

Reply:
Noted, will look into more later when I have more time.